apps/fyle/signals.py

```
@@ -11,7 +10,8 @@
 
 from apps.fyle.models import ExpenseFilter, ExpenseGroupSettings
 from apps.fyle.tasks import re_run_skip_export_rule
-from apps.workspaces.models import Configuration
+from apps.workspaces.models import Configuration, Workspace
```

Ruff (0.12.2)

`apps.workspaces.models.Workspace` imported but unused

Remove unused import: `apps.workspaces.models.Workspace`

(F401)